doc/introduction/measurements.rst

```diff
@@ -145,3 +175,44 @@ number of shots. This can be done by modifying the value of :attr:`.Device.shots
     # execute the QNode again, now using 1 shot
     dev.shots = 1
     result = circuit(0.54, 0.1)
+
+
+With an increasing number of shots, the average over
+measurement samples converges to the exact expectation of an observable. Consider the following
+circuit:
+
+.. code-block:: python
+
+    from pennylane import numpy as np
+
+    # fix seed to produce same samples in different runs
+    np.random.seed(1)
+
+    dev = qml.device("default.qubit", wires=1)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Hadamard(wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+Running the simulator without specifying the shot number returns the exact expectation.
+
+>>> circuit()
+0.0
+
+Now we set the device to return stochastic results, and increase the shot number.
```

Comment:
```suggestion
Now we set the device to return stochastic results, and increase the number of shots starting from ``10``.
```

Reply:
(The suggestion mainly aims at rephrasing based on my previous comment referring to ``shots=1000`` being the default, but anything else could work as well here! :) )